docs/lcb/notes/22060.md

```diff
@@ -0,0 +1,4 @@
+# LiveCode Builder Host Library
+## Canvas library
+
+# Fix bug in canvas library that can result in Points being initialized with previously used values.
```

Comment:
@livecodeian I think there is an errant `#` here which will stuff up the release notes parsing

Reply:
@livecodeian oh I think you may just want to make it like this so there's just a bugfix note
```
# [22060] Fix bug in canvas library that can result in Points being initialized with previously used values.
```

Reply:
@livecodeian I'm fairly sure it needs the `[22060]` bit to figure out the bug number